dune-project

```diff
@@ -1,4 +1,4 @@
-(lang dune 3.6)
+(lang dune 3.7)
 (name coq)
 
 ; We use directory targets in documentation
```

Comment:
Can you also bump the coq lang as far as it will go? It should probably be around 0.7.

Reply:
Indeed, 0.7

Reply:
Note that our use of coq-lang is almost anecdotal, it is only used to have `(coq.pp ...)` declarations, that otherwise we could write very easily by hand.

On the other hand, there was hope we could build Coq with dune fully at some point, indeed, as of today using `theories/dune.disabled` works pretty well.

If we were to declare a `(lang coq ...)` version, 0.8 would be the preferred one, as it is the candidate for `(lang coq 1.0)`